packages/cli/fern-definition/ir-to-jsonschema/src/__test__/__snapshots__/alias/type__Type.json

```diff
@@ -2,10 +2,12 @@
   "type": "object",
   "properties": {
     "id": {
-      "$ref": "#/definitions/TypeId"
+      "$ref": "#/definitions/TypeId",
+      "title": "id"
```

Comment:
@pattobrien thoughts on making the `title` the actual name of the type so `TypeId` ?

Reply:
Resolving, based on other comment thread.